examples/profile-subscription.yaml

```diff
@@ -5,4 +5,4 @@ metadata:
   namespace: default
 spec:
   profileURL: https://github.com/weaveworks/nginx-profile
-  branch: main
+  branch: move-helm-to-nested
```

Comment:
TODO revert back to main when https://github.com/weaveworks/nginx-profile/pull/7 is merged